programs/scope/src/lib.rs

```diff
@@ -133,6 +151,50 @@ impl Default for DatedPrice {
     }
 }
 
+#[zero_copy]
+#[derive(Debug, Eq, PartialEq)]
+pub struct EmaTwap {
+    pub current_ema_1h: u128,
+
+    pub last_update_slot: u64, // the slot when the last observation was added
+    pub last_update_unix_timestamp: u64,
+
+    pub padding: [u128; 40],
+}
```

Comment:
Just to group the emas of different period
```suggestion
pub struct EmaTwap {
    pub last_update_slot: u64, // the slot when the last observation was added
    pub last_update_unix_timestamp: u64,

    pub current_ema_1h: u128,

    pub padding: [u128; 40],
}
```